src/lava/lib/optimization/solvers/generic/dataclasses.py

```diff
@@ -73,6 +93,10 @@ def local_cost(self):
     def variables_assignment(self):
         return self.discrete.variable_assignment
 
+    @property
+    def variables_assignment_cont(self):

```

Comment:
What does _cont refer to? For me, this looks more like a def variable_assignmende than the actual def variable_assignment above.

Reply:
continous